tests/test_system_models.py

```
@@ -0,0 +1,133 @@
+from pydantic import ValidationError
+
+from isminet.models.system import (
+    SystemHealth,
+    ProcessInfo,
+    ServiceStatus,
+    SystemStatus,
+)
+from isminet.models.enums import DeviceType
+
+# Test data
+VALID_SYSTEM_HEALTH: Dict[str, Any] = {
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)